neuralmonkey/dataset/lazy_dataset.py

```diff
@@ -27,30 +27,46 @@ def __init__(self, name: str,
                 ) -> None:
         """Create a new instance of the lazy dataset.
 
+        Lazy dataset first initializes the parent `Dataset` object with empty
+        series
+
         Arguments:
-            name: The name of the dataset series_paths_and_readers: The mapping
-            of series name to its file series_outputs: Dictionary mapping
-            series names to their output file preprocess: The preprocessor to
-            apply to the read lines
+            name: The name of the dataset
+            series_paths_and_readers: Dictionary that maps each series ID to a
+                list of files and a reader.
+            series_outputs: A mapping of series IDs to output files.
+            preprocessors: The preprocessors to apply to the data series.
+                Each preprocessor is defined by soruce series ID, preprocessed
```

Comment:
soruce -> source
preprocessed -> resulting preprocessed (aby bylo jasne, ze to vznikne tim preprocessingem)

Reply:
jasný